source/_components/canary.markdown

```diff
@@ -0,0 +1,41 @@
+---
+layout: page
+title: "Canary"
+description: "Instructions on how to integrate your Canary devices into Home Assistant."
+date: 2017-12-07 22:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: canary.png
+ha_category: Hub
+ha_release: "0.60"
+ha_iot_class: "Cloud Polling"
+---
+
+The `canary` component allows you to integrate your [Canary](https://canary.is) devices in Home Assistant.
+
+You will need your Canary login information (username, usually you email address, and password) to use this module.
+
+To set it up, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+canary:
+  username: you@example.com
+  password: secret
+```
+
+Configuration variables:
```

Comment:
Please replace this section with the configuration tag and use that syntax to mark the options:
https://home-assistant.io/developers/documentation/standards/#component-and-platform-pages